index.js

```diff
@@ -49,6 +49,10 @@ function copy(src) {
 			dst = { message: src.message };
 		} else if (isBoolean(src) || isNumber(src) || isString(src)) {
 			dst = Object(src);
+		} else if (src instanceof Buffer) {
```

Comment:
instanceof isn't reliable; this would need to use Buffer.isBuffer

Reply:
Good point, will modify this to `(typeof Buffer !== 'undefined' && typeof Buffer.isBuffer === 'function' && Buffer.isBuffer(src))`